src/screenshot_gui.cpp

```diff
@@ -0,0 +1,112 @@
+/* $Id$ */
+
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ /** @file screenshot_gui.cpp GUI functions related to screenshots. */
+
+#include "stdafx.h"
+#include "gui.h"
+#include "viewport_func.h"
+#include "window_func.h"
+#include "window_gui.h"
+#include "screenshot.h"
+#include "textbuf_gui.h"
+
+#include "table/strings.h"
+
+enum ScreenshotWindowWidgets {
+	WID_SC_TAKE,
+	WID_SC_TAKE_CRASHLOG,
+	WID_SC_TAKE_ZOOMIN,
+	WID_SC_TAKE_DEFAULTZOOM,
+	WID_SC_TAKE_WORLD,
+	WID_SC_TAKE_HEIGHTMAP,
+};
```

Comment:
The widgets need to be defined in a separate header file and have appropriate Doxygen comments, so they can be picked up by the script API generator. (Even if I can't imagine anyone ever needing to do any tutorial for the screenshot window.)